src/__tests__/index.test.ts

```diff
@@ -206,6 +206,13 @@ const vectors: Array<TestVector> = [
     passingVectors: [
       { text: 'cosmos1depk54cuajgkzea6zpgkq36tnjwdzv4afc3d27', hex: '6e436a571cec916167ba105160474b9c9cd132bd' }
     ],
+  },
+  {
+    name: 'ADA',
+    coinType: 1815,
+    passingVectors: [
+      { text: 'addr1gqtnpvdhqrtpd4g424fcaq7k0ufuzyadt7djygf8qdyzevuph3wczvf2dwyx5u', hex: '401730b1b700d616d51555538e83d67f13c113ad5f9b22212703482cb381bc5d81312a' }
+    ],
```

Comment:
It seems that ADA has `addr` prefix for mainnet and `addr_test` for testnet 
https://github.com/cardano-foundation/CIPs/pull/6/files#diff-50134d380e035316e17c63951fe2d813R33

@Arachnid do we enforce the support of test address as well?

Reply:
@makoto Do we want the support for test address as well?